build/protractor.conf.js

```diff
@@ -43,14 +29,12 @@
   exports.config = {
 
     suites: {
-      all: '../test/e2e/tests/**/*.spec.js',
-      localhost: '../test/e2e/tests/localhost/**/*.spec.js',
-      other: '../test/e2e/tests/other/**/*.spec.js',
-      acceptance: acceptanceTests
+      all: '../components/**/frontend/test/e2e/**/*.spec.js',
```

Comment:
Should we remove all? Is it used? It would only work for those components that have a front-end folder.